src/opera_disp_tms/__main__.py

```diff
@@ -53,13 +53,19 @@ def generate_mosaic_geotiff(
 def generate_tile_map_service(
     tile_type: str, bbox: tuple[int, int, int, int], direction: str, begin_date: datetime, end_date: datetime
 ) -> Path:
-    mosaic = generate_mosaic_geotiff(tile_type, bbox, direction, begin_date, end_date)
+    mosaic_geotiffs = []
+    for partition in partition_bbox(bbox):
+        try:
```

Comment:
I'm curious about the `try` block here. If a single 10x10 block fails, I think we want the whole job to fail.

Reply:
It's for large aois that don't have contiguous data coverage. I kicked off a job for the whole western hemisphere yesterday, which includes many 10x10 partitions with no data (e.g. over ocean). I want to skip those but still create georiffs for the partitions that do include data, then mosaic them.